content/en/docs/installation/supported-releases.md

```diff
@@ -207,7 +207,7 @@ Our testing coverage is:
 | Release branch | Prow configuration            | testgrid                | Kubernetes versions tested   | Periodicity |
 | :------------: | :---------------------------- | :---------------------- | :--------------------------- | :---------: |
 |     master     | [`next-periodics.yaml`][]     | [`testgrid-next`][]     | 1.16, 1.17, 1.18, 1.19, 1.20 | On each PR  |
-|  release-1.3   | [`previous-periodics.yaml`][] | [`testgrid-previous`][] | 1.16, 1.17, 1.18, 1.19       | Every night |
+|  release-1.3   | [`previous-periodics.yaml`][] | [`testgrid-previous`][] | 1.16, 1.17, 1.18, 1.19, 1.20 | Every night |
```

Comment:
> presubmits.yaml seems to be running Kubernetes 1.16 to 1.20 (not 1.21 though)... not sure though, I don't fully understand how the Prow config works sweat_smile

Good catch- you're right and it also made me realize that v1.16- v1.19 presubmits seem to not have run since April, see i.e https://testgrid.k8s.io/jetstack-cert-manager-presubmits-blocking#pull-cert-manager-e2e-v1-17. I am not entirely sure if we do want to run those. I can look into this.

Reply:
Actually, we explicitly don't run v1.16- v1.19 on every PR, see https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/cert-manager-presubmits.yaml#L149

Reply:
Woah thanks for bringing that up, I had not noticed the \`always\_run: false\`!!

I'll fix the table 👍